truss/remote/baseten/api.py

```diff
@@ -213,6 +249,37 @@ def deploy_chain(self, name: str, chainlet_data: List[b10_types.ChainletData]):
         resp = self._post_graphql_query(query_string)
         return resp["data"]["deploy_chain"]
 
+    def deploy_chain_atomic(
```

Comment:
A very high level comment: we have only one push command that handles all cases of draf/non-draft and new root entity creation and addition of a deploymen to an existing entity.

But then we have 3 client functions and 3 GQL endpoints each with a lot of boilerplate code, argument type definitions... etc - can we not fold all of this into one GQL endpoint?